WordPress/src/main/java/org/wordpress/android/viewmodel/pages/CreatePageListItemLabelsUseCase.kt

```diff
@@ -24,19 +24,25 @@ import org.wordpress.android.viewmodel.pages.CreatePageUploadUiStateUseCase.Post
 import org.wordpress.android.viewmodel.pages.CreatePageUploadUiStateUseCase.PostUploadUiState.UploadingPost
 import javax.inject.Inject
 
+typealias LabelColor = Int?
 /**
  * Most of this code has been copied from PostListItemUIStateHelper.
  */
-class CreatePageListItemLabelsUseCase @Inject constructor(private val pageConflictResolver: PageConflictResolver) {
-    fun createLabels(pagePostModel: PostModel, uploadUiState: PostUploadUiState): List<UiString> {
-        return getLabels(
+class CreatePageListItemLabelsUseCase @Inject constructor(
+    private val pageConflictResolver: PageConflictResolver,
+    private val labelColorUseCase: ResolvePageListItemsColorUseCase
+) {
+    fun createLabels(pagePostModel: PostModel, uploadUiState: PostUploadUiState): Pair<List<UiString>, LabelColor> {
```

Comment:
Should this `LabelColor` be nullable as well? 
```suggestion
    fun createLabels(pagePostModel: PostModel, uploadUiState: PostUploadUiState): Pair<List<UiString>, LabelColor?> {
```

Reply:
LabelColor is a typealias for nullable integer -> `Int?`. So I believe it is nullable. Good question though:)!

Reply:
Oh I just saw it! 😅